tests/test_translator.py

```diff
@@ -102,8 +103,8 @@ def test_token(token, result):
 @pytest.mark.parametrize("token", ["ERROR_ON_FILE", "ERROR_ON_FOLDER", "ERROR_ON_BOTH"])
 def test_truncated_paths(token):
```

Comment:
This test can be moved to `test_utils.py` as the `Translator` does not handle the short name creation anymore.